frontend/src/routes/manage/Video/Access.tsx

```diff
@@ -111,7 +111,10 @@ const AccessUI: React.FC<AccessUIProps> = ({ event, knownRoles }) => {
     const saveModalRef = useRef<ConfirmationModalHandle>(null);
     const [commitError, setCommitError] = useState<JSX.Element | null>(null);
     const [commit, inFlight] = useMutation<AccessUpdateAclMutation>(updateVideoAcl);
-    const [editingBlocked, setEditingBlocked] = useState(event.hasActiveWorkflows);
+    const aclLockedToSeries = CONFIG.lockAclToSeries && event.series;
```

Comment:
Again: do we also want to check whether the series even has ACL? And unlock it if it hasn't? :thinking:

Reply:
Yep, that makes sense to me! Then I guess just removing this one check from the uploader